code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/impl/ApiAuthenticator.java

```diff
@@ -131,10 +134,25 @@ protected Account getAccount(ApiRequest request) {
                 }
             }
         } else if (!StringUtils.equals("true", authHeader)) {
-            account = authDao.getAdminAccount();
+            Account admin = authDao.getAdminAccount();
```

Comment:
I will need  to talk to @ibuildthecloud and @vincent99 as to the behavior of this method ill find out.